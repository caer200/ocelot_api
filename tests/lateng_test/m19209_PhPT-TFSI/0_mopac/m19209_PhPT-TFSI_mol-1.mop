CHARGES

 
H	1.886021 +1 12.577714 +1 5.419329 +1
C	1.834352 +1 12.459189 +1 4.478095 +1
C	1.764071 +1 13.588384 +1 3.635233 +1
C	1.828958 +1 11.190545 +1 3.952192 +1
C	1.665910 +1 13.374235 +1 2.241977 +1
N	1.807550 +1 14.860807 +1 4.184574 +1
H	1.886153 +1 10.440960 +1 4.533118 +1
C	1.739759 +1 10.989245 +1 2.566091 +1
C	1.656534 +1 12.066792 +1 1.727878 +1
S	1.561526 +1 14.642375 +1 1.091614 +1
C	1.732623 +1 16.043161 +1 3.463892 +1
C	2.055400 +1 14.959945 +1 5.616805 +1
H	1.737294 +1 10.107984 +1 2.211420 +1
H	1.591730 +1 11.930001 +1 0.790111 +1
C	1.612308 +1 16.061049 +1 2.055980 +1
C	1.788964 +1 17.280691 +1 4.136747 +1
C	0.985755 +1 15.077601 +1 6.485880 +1
C	3.365343 +1 14.932862 +1 6.056298 +1
C	1.545180 +1 17.280817 +1 1.366653 +1
H	1.868114 +1 17.296386 +1 5.083236 +1
C	1.730963 +1 18.463297 +1 3.442190 +1
H	0.092975 +1 15.077261 +1 6.161018 +1
C	1.248375 +1 15.196517 +1 7.847686 +1
H	4.083260 +1 14.841836 +1 5.440903 +1
C	3.607135 +1 15.041952 +1 7.421817 +1
H	1.452247 +1 17.282681 +1 0.421242 +1
C	1.612391 +1 18.471485 +1 2.045487 +1
H	1.771779 +1 19.286687 +1 3.914270 +1
H	0.530142 +1 15.284797 +1 8.463123 +1
C	2.553588 +1 15.186313 +1 8.306376 +1
H	4.499036 +1 15.017249 +1 7.748123 +1
H	1.578810 +1 19.294749 +1 1.572645 +1
H	2.727481 +1 15.279128 +1 9.235707 +1
