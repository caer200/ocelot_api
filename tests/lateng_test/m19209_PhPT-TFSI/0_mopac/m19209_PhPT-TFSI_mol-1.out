 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:29:54 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          1.88602100  *  12.57771400  *   5.41932900  *
     2       C          1.83435200  *  12.45918900  *   4.47809500  *
     3       C          1.76407100  *  13.58838400  *   3.63523300  *
     4       C          1.82895800  *  11.19054500  *   3.95219200  *
     5       C          1.66591000  *  13.37423500  *   2.24197700  *
     6       N          1.80755000  *  14.86080700  *   4.18457400  *
     7       H          1.88615300  *  10.44096000  *   4.53311800  *
     8       C          1.73975900  *  10.98924500  *   2.56609100  *
     9       C          1.65653400  *  12.06679200  *   1.72787800  *
    10       S          1.56152600  *  14.64237500  *   1.09161400  *
    11       C          1.73262300  *  16.04316100  *   3.46389200  *
    12       C          2.05540000  *  14.95994500  *   5.61680500  *
    13       H          1.73729400  *  10.10798400  *   2.21142000  *
    14       H          1.59173000  *  11.93000100  *   0.79011100  *
    15       C          1.61230800  *  16.06104900  *   2.05598000  *
    16       C          1.78896400  *  17.28069100  *   4.13674700  *
    17       C          0.98575500  *  15.07760100  *   6.48588000  *
    18       C          3.36534300  *  14.93286200  *   6.05629800  *
    19       C          1.54518000  *  17.28081700  *   1.36665300  *
    20       H          1.86811400  *  17.29638600  *   5.08323600  *
    21       C          1.73096300  *  18.46329700  *   3.44219000  *
    22       H          0.09297500  *  15.07726100  *   6.16101800  *
    23       C          1.24837500  *  15.19651700  *   7.84768600  *
    24       H          4.08326000  *  14.84183600  *   5.44090300  *
    25       C          3.60713500  *  15.04195200  *   7.42181700  *
    26       H          1.45224700  *  17.28268100  *   0.42124200  *
    27       C          1.61239100  *  18.47148500  *   2.04548700  *
    28       H          1.77177900  *  19.28668700  *   3.91427000  *
    29       H          0.53014200  *  15.28479700  *   8.46312300  *
    30       C          2.55358800  *  15.18631300  *   8.30637600  *
    31       H          4.49903600  *  15.01724900  *   7.74812300  *
    32       H          1.57881000  *  19.29474900  *   1.57264500  *
    33       H          2.72748100  *  15.27912800  *   9.23570700  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          1.8860     12.5777      5.4193
     2         C          1.8344     12.4592      4.4781
     3         C          1.7641     13.5884      3.6352
     4         C          1.8290     11.1905      3.9522
     5         C          1.6659     13.3742      2.2420
     6         N          1.8075     14.8608      4.1846
     7         H          1.8862     10.4410      4.5331
     8         C          1.7398     10.9892      2.5661
     9         C          1.6565     12.0668      1.7279
    10         S          1.5615     14.6424      1.0916
    11         C          1.7326     16.0432      3.4639
    12         C          2.0554     14.9599      5.6168
    13         H          1.7373     10.1080      2.2114
    14         H          1.5917     11.9300      0.7901
    15         C          1.6123     16.0610      2.0560
    16         C          1.7890     17.2807      4.1367
    17         C          0.9858     15.0776      6.4859
    18         C          3.3653     14.9329      6.0563
    19         C          1.5452     17.2808      1.3667
    20         H          1.8681     17.2964      5.0832
    21         C          1.7310     18.4633      3.4422
    22         H          0.0930     15.0773      6.1610
    23         C          1.2484     15.1965      7.8477
    24         H          4.0833     14.8418      5.4409
    25         C          3.6071     15.0420      7.4218
    26         H          1.4522     17.2827      0.4212
    27         C          1.6124     18.4715      2.0455
    28         H          1.7718     19.2867      3.9143
    29         H          0.5301     15.2848      8.4631
    30         C          2.5536     15.1863      8.3064
    31         H          4.4990     15.0172      7.7481
    32         H          1.5788     19.2947      1.5726
    33         H          2.7275     15.2791      9.2357

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C18 H13 N S  =    33 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
